### PR TITLE
[MIG] hotel_reservation: Migration to 11.0

### PR DESCRIPTION
**Pull request by KrishnaPrajapati-SerpentCS**

### CODE REVIEW

File: hotel_reservation/README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==================
+Hotel Reservation
+==================
```
Comment (JayVora-SerpentCS) [inline hotel_reservation/README.rst:7]: Align = with n.

File: hotel_reservation/README.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+.. image:: https://raw.githubusercontent.com/OCA/vertical-hotel/10.0/hotel_reservation/static/description/res6.png
+   :width: 750px
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/157/10.0
```
Comment (JayVora-SerpentCS) [inline hotel_reservation/README.rst:52]: 11.0

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by serpentcs-dev1** [comment]
Supersedes Now with #42

File: hotel_reservation/static/src/js/hotel_room_summary.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+         this.summary_header = py.eval(this.get("summary_header"));
+        }
+        if (this.get("room_summary")) {
+         this.room_summary = py.eval(this.get("room_summary"));
+        }
+        this.renderElement();
+        this.view_loading();
+     },
+     initialize_field: function() {
+         form_common.ReinitializeWidgetMixin.initialize_field.call(this);
```
Comment (vbezl) [inline hotel_reservation/static/src/js/hotel_room_summary.js:73]: no declaration of form_common

File: hotel_reservation/static/src/js/hotel_room_summary.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            });
+        this.set({
+            date_to: false,
+            date_from: false,
+            summary_header: false,
+            room_summary: false,
+        });
+        this.summary_header = [];
+        this.room_summary = [];
+        this.on("_onFieldChanged:date_from", this, function() {
```
Comment (vbezl) [inline hotel_reservation/static/src/js/hotel_room_summary.js:38]: this doesn't work for some reason - nothing is printed to console when changing date_from

**Pull request closed without merging**